worldedit-core/src/main/resources/lang/.gitignore

```diff
@@ -0,0 +1 @@
+*/
```

Comment:
This seems out of place. What's the issue with putting this rule in the base gitignore?